assets/css/editor.css

```diff
@@ -175,10 +190,10 @@ Also some spacing adjustments.
   border-radius: 2px;
   content: "";
   display: block;
-  height: 12px;
-  width: 12px;
+  height: var(--decoration-size);
+  width: var(--decoration-size);
   position: absolute;
-  top: 50%;
+  top: calc(50% - 1px);
```

Comment:
It was one of those situations where, though it _was_ in the middle, it looks visibly misaligned at 50% because the font doesn't render perfectly centered. I'm fine reverting it though.

**With the 1px offset:**

![image](https://github.com/livebook-dev/livebook/assets/503938/fe27954a-fb35-4d93-991e-8e8930213951)
![image](https://github.com/livebook-dev/livebook/assets/503938/7f30d436-3365-4fe1-9193-a95e5c789924)


**Without the 1px offset:**

![image](https://github.com/livebook-dev/livebook/assets/503938/69f75727-5174-4cc2-aa41-a4bf1ad824d4)
![image](https://github.com/livebook-dev/livebook/assets/503938/cb665362-426e-42a0-ac14-e47f3fb21235)

Reply:
The impression is probably because `i` is slightly to the top, if you select the whole line and compare with selection it's more clear. Also if you zoom the page it's more evident :)

Reply:
Maybe it’s a rendering thing — on my screen, it certainly looks like the default is misaligned compared to the 1px offset, but I don’t feel strongly about it! On my phone now but will revert this change on my next pass!

![5DBF09B8-0CAA-4401-8552-7F8D80C32B11](https://github.com/livebook-dev/livebook/assets/503938/09fe0ec0-9a38-45ab-af9f-a70069638c82)

![78E08F75-3574-4E4B-B682-31CE2FFCF7FB](https://github.com/livebook-dev/livebook/assets/503938/9e5f7691-71bb-4dee-937b-bf8a610b9135)

Reply:
Zooming 150%, with offset:

<img width="257" alt="image" src="https://github.com/livebook-dev/livebook/assets/17034772/e1f39061-493b-48eb-96df-518ccf437344">

without:

<img width="253" alt="image" src="https://github.com/livebook-dev/livebook/assets/17034772/9bc68e2c-adfc-4845-9661-abe62f80804e">

:D